src/examples/kotlin/rx/lang/kotlin/examples/examples.kt

```diff
@@ -1,115 +1,119 @@
 package rx.lang.kotlin.examples
 
-import rx.Observable
-import rx.lang.kotlin.*
-import rx.subscriptions.CompositeSubscription
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import rx.lang.kotlin.FunctionSubscriber
+import rx.lang.kotlin.addTo
+import rx.lang.kotlin.combineLatest
+import rx.lang.kotlin.observable
+import rx.lang.kotlin.plusAssign
+import rx.lang.kotlin.subscribeBy
+import rx.lang.kotlin.toObservable
+import rx.lang.kotlin.zip
 import java.net.URL
-import java.util.*
+import java.util.Scanner
 import java.util.concurrent.TimeUnit
 import kotlin.concurrent.thread
 
 fun main(args: Array<String>) {
-
-    val subscription = CompositeSubscription()
-
+    
+    val subscription = CompositeDisposable()
+    
     val printArticle = { art: String ->
         println("--- Article ---\n${art.substring(0, 125)}")
     }
-
+    
     val printIt = { it: String -> println(it) }
-
+    
     subscription += asyncObservable().subscribe(printIt)
-
+    
     subscription += syncObservable().subscribe(printIt)
-
+    
     subscription.clear()
-
+    
     simpleComposition()
-
+    
     asyncWiki("Tiger", "Elephant").subscribe(printArticle)
-
+    
     asyncWikiWithErrorHandling("Tiger", "Elephant").subscribe(printArticle) { e ->
         println("--- Error ---\n${e.message}")
     }
-
+    
```

Comment:
Looks like default kotlin formatting settings in one of the last plugin releases.